### PR TITLE
[RLlib] Enhance CI (increase parallelism; remove tf2-eager-tracing learning tests)

### PR DESCRIPTION
**Pull request by sven1977**
Enhance CI (increase parallelism; remove tf2-eager-tracing learning tests)

## Checks

- [x] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: .buildkite/rllib.rayci.yml [hunk 1]
```diff
@@ -189,13 +177,13 @@ steps:
...
       - gpu
-    parallelism: 2
+    parallelism: 4
     instance_type: gpu-large
     commands:
       - bazel run //ci/ray_ci:test_in_docker -- //rllib/... rllib
         --workers "$${BUILDKITE_PARALLEL_JOB_COUNT}" --worker-id "$${BUILDKITE_PARALLEL_JOB}"
         --parallelism-per-worker 2
-        --gpus 4
+        --gpus 8
```
Comment (can-anyscale) [inline .buildkite/rllib.rayci.yml:186]: this should be 4 still

**Comment by can-anyscale** [comment]
and delete this test as well https://github.com/ray-project/ray/blob/master/ci/ray_ci/test_linux_tester_container.py#L82, sorry about this

File: .buildkite/rllib.rayci.yml (hunk 1 above)
Comment (sven1977) [reply .buildkite/rllib.rayci.yml]: fixed

**Review by simonsays1980 — APPROVED** [review]
LGTM. Does this also remove the `tf2` tests for the old stack CQL/SAC?

**Comment by sven1977** [comment]
> LGTM. Does this also remove the `tf2` tests for the old stack CQL/SAC?

We only remove tf2 + eager tracing tests here.
tf2 + static-graph tests remain in place for now ...

**Comment by sven1977** [comment]
> and delete this test as well https://github.com/ray-project/ray/blob/master/ci/ray_ci/test_linux_tester_container.py#L82, sorry about this

Done.

**Review by can-anyscale — APPROVED** [review]

**Pull request merged**